internal/test/wmcb/wmcb_test.go

```diff
@@ -614,3 +645,69 @@ func testWMCBCluster(t *testing.T) {
 	require.NoErrorf(t, err, "error while getting Windows node: %v", err)
 	assert.Lenf(t, winNodes.Items, 1, "expected one node to have node label but found: %v", len(winNodes.Items))
 }
+
+// initializeCNIPluginsPkgInfo populates the fields of cniPlugins
+func (f *wmcbFramework) initializeCniPluginsPkgInfo() error {
+	cniPlugins.url = framework.GetLatestCniPluginsURL()
+
+	cniPluginsSHA, err := framework.GetLatestCniPluginsSHA()
+	if err != nil {
+		return fmt.Errorf("unable to get SHA for CNI Plugins: %v", err)
+	}
+	cniPlugins.sha = cniPluginsSHA
+
+	cniPlugins.shaType = "sha512"
+	return nil
+}
+
+// getLatestCniPluginsVersion returns the latest 0.8.x version of CNI plugins in 'v<major>.<minor>.<patch>' format
+func (f *wmcbFramework) getLatestCniPluginsVersion() error {
+	releases, err := f.TestFramework.GetGithubReleases("containernetworking", "plugins")
+	if err != nil {
+		return err
+	}
+	// Iterating over releases to fetch versions from tag names
+	var cniPluginsVersions = []string{}
+	for _, release := range releases {
+		cniPluginsVersions = append(cniPluginsVersions, release.GetTagName())
+	}
+	// Sorting versions in reverse order so as to get latest version first
+	sort.Sort(sort.Reverse(sort.StringSlice(cniPluginsVersions)))
+	// Iterating over versions to find first 0.8.x version which is not a release candidate
+	for _, cniPluginsVersion := range cniPluginsVersions {
+		if strings.HasPrefix(cniPluginsVersion, "v0.8.") && !strings.Contains(cniPluginsVersion, "rc") {
+			f.LatestCniPluginsVersion = cniPluginsVersion
+			return nil
+		}
+	}
+	return fmt.Errorf("could not fetch latest 0.8.x version of CNI Plugins")
+}
+
+// GetLatestCniPluginsURL returns the URL of the latest 0.8.x version of CNI Plugins
+func (f *wmcbFramework) GetLatestCniPluginsURL() string {
```

Comment:
Why do we need a method for this and a public one

Reply:
nit and purely for fun comment: Please start using variables and we're not in a purely functional language like racket or scheme

Reply:
Removed this function